emcc.py

```diff
@@ -1803,7 +1802,6 @@ def include_and_export(name):
         not shared.Settings.RELOCATABLE and \
         not target.endswith(WASM_ENDINGS) and \
         not shared.Settings.ASYNCIFY_LAZY_LOAD_CODE and \
-        not shared.Settings.WASM2JS and \
```

Comment:
is this properly rebased? this change for example landed in a PR of mine recently, didn't it?

Reply:
That is not part of the diff (at least I don't see it in the UI right now).